### DISCUSSION TITLE
part-ii/2021-6/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#359] [Question comments]
Part II 2021 Algebraic Topology: Paper 3, Section II, 20F 

https://questions.tripos.org/part-ii/2021-6/

**Comment by artie2000** [comment]
<details>
<summary>(a)</summary>

Triangulate $CX$ using $Cone(K)$, where $K$ is a triangulation of $X$. Then the homology vanishes (except at 0).

</details>

(b) is bookwork.

<details>
<summary>(c)</summary>

We need to use that the inclusion of $A$ into $X$ maps the generator of $H_1(A)$ to one of the two generators of $H_1(X)$ (in some basis). 

This is clear geometrically; maybe a way to do it rigorously is to carefully choose a triangulation, but I don't think it's that important.

The homology is:

$$H_i(X')=\begin{cases}\mathbb{Z}&i\in\{0,1,2\}\\0&\text{o/w}\end{cases}$$

</details>
